src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -114,22 +129,10 @@ function showVisibilityCheckboxesIfCustomOptionSelected($containingForm) {
     }
 }
 
-var checkCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === false) {
-        $(checkbox).trigger('change');
-    }
-    checkbox.checked = true;
-};
-
-var uncheckCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === true) {
-        $(checkbox).trigger('change');
-    }
```

Comment:
@wkurniawan07 The problem with the previous code (i.e., wrong visibility message displayed because AJAX response returned out of order) was that the change was triggered multiple times and thus sending multiple AJAX requests whenever a common option is selected, so I removed the triggering of change at per checkbox level (i.e., the removed block), and rely on the final trigger that actually already existed.